app/src/test/java/com/esoxjem/movieguide/util/RxSchedulersOverrideRule.java

```diff
@@ -4,47 +4,18 @@
 import org.junit.runner.Description;
 import org.junit.runners.model.Statement;
 
-import rx.Scheduler;
-import rx.android.plugins.RxAndroidPlugins;
-import rx.android.plugins.RxAndroidSchedulersHook;
-import rx.functions.Func1;
-import rx.plugins.RxJavaHooks;
-import rx.schedulers.Schedulers;
-
 /**
  * This rule registers SchedulerHooks for RxJava and RxAndroid to ensure that subscriptions
  * always subscribeOn and observeOn Schedulers.immediate().
  * Warning, this rule will reset RxAndroidPlugins and RxJavaPlugins before and after each test so
  * if the application code uses RxJava plugins this may affect the behaviour of the testing method.
  */
 public class RxSchedulersOverrideRule implements TestRule {
```

Comment:
Looks like this rule isn't required anymore? Please delete this class.

Reply:
Ok, I was waiting for confirmation on this, I ll do that